HaulageApplication/HaulageTests/AdministratorTests.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+using Haulage.BaseClasses.Accounting;
+
+namespace HaulageTests
+{
+    public class AdministratorTests
+    {
+        [Fact]
+        public void AddManifestItem_ShouldAddItemToManifest()
+        {
+            // Arrange
+            var admin = new Administrator();
+            var manifest = new TripManifest();
+            var item = new ManifestItem(1, "Item 1", "Other");
+
+            // Act
+            admin.AddManifestItem(manifest, item);
+
+            // Assert
+            Assert.Contains(item, manifest.Items);
+        }
+
+        [Fact]
+        public void RemoveManifestItem_ShouldRemoveItemFromManifest()
+        {
+            // Arrange
+            var admin = new Administrator();
+            var manifest = new TripManifest();
+            var item = new ManifestItem(1, "Item 1", "Other");
+            manifest.Items.Add(item);
+
+            // Act
+            admin.RemoveManifestItem(manifest, item);
+
+            // Assert
+            Assert.DoesNotContain(item, manifest.Items);
+        }
+
+        [Fact]
+        public void UpdateManifestItem_ShouldUpdateItemInManifest()
```

Comment:
Should there also be a test for the case where the old item does not exist in the manifest and you try to update it with a new item?

Reply:
LJ - That is not planned to be a feature but will add as future development task.